src/index.js

```diff
@@ -14,7 +14,11 @@ let tryRequire = (pathOrId: string | number) => {
   try {
     // $FlowIgnore
     return babelInterop(require(pathOrId));
-  } catch (err) {}
+  } catch (error) {
+    if (error.name === "SyntaxError") {
```

Comment:
Maybe instead we can exclusively catch only the exact errors which will be throw by node/webpack/browserify/etc when `require()`-ing something invalid.